weatherman.py

```diff
@@ -0,0 +1,80 @@
+"""
+Generating reports of temperature from given text files.
+"""
+
+import os
+import argparse
+import sys
+
+file_list = os.listdir('./weatherdata/')
```

Comment:
Program does not use argument given by user and uses this hard coded directory address to read files

Reply:
Done, but have a follow up question about this.